src/ib_module.F

```diff
@@ -1680,7 +1680,7 @@ subroutine ib_lwr_flx(stag,ix,jy,kz,c1,c2,rrw,dumz,a,kbdy,vadvorder)
         j2=nj+1
       endif
 
-    !$acc parallel loop private(i,j,k,kval,wbar,cc1,cc2)
+    !$acc parallel loop default(present) private(i,j,k,kval,wbar,cc1,cc2)
```

Comment:
maybe add `gang vector` clause and `collapse` clause, and remove `private` clause?